### ISSUE TITLE
Entity Item Counts

### ISSUE THREAD

**Issue by tn3rb** [eventespresso/event-espresso-core#2998]
### issue posted by @garthkoyle 

Why is there a "Related Ticket" quantity bubble when there are no tickets visible in the Tickets section? https://www.screencast.com/t/jVFXfByf

I noticed that when I saved an event as a draft, the tickets appeared on the screen, but again not until I saved the event as a draft.

Why did it create two tickets, with different names "B" and different quantities? Maybe there is another setting I'm missing that is creating these? I checked the default pricing and didn't see any there: https://www.screencast.com/t/Lbt7wMmSt9H, https://brent.eventespresso.com/wp-admin/admin.php?page=pricing&action=default&default_nonce=e7fb03457a&return=price_types

**Comment by tn3rb** [comment]
> Why is there a "Related Ticket" quantity bubble when there are no tickets visible in the Tickets section?

there's two answers to this question:

- ### general answer
what tickets appear in the tickets list depends on what your current filter settings are and not whether dates above have related tickets. So for example, if your two tickets were both `pending`, but your filters for the ticket list were set to `expired`, then you are not going to see those tickets in the tickets list regardless of what dates they are related to.

- ### specific answer to what I think the situation is that you encountered
there's a tooltip showing in your screenshot that says "Add New Event..." so I went and checked what happens on the dev test server and it looks like those initial tickets do NOT show up in the tickets list (despite the filter settings being ok) when you first create an event. Created the following issue to fix this: https://github.com/eventespresso/event-espresso-core/issues/2999

> Why did it create two tickets, with different names "B" and different quantities?

Because that testing site has the following two default tickets:

![ScreenShot_20200624131359](https://user-images.githubusercontent.com/1751030/85624710-2ccb8c00-b61f-11ea-9e44-359401821316.png)

![ScreenShot_20200624131424](https://user-images.githubusercontent.com/1751030/85624747-37862100-b61f-11ea-8cbf-2b2e593398fc.png)

>  I checked the default pricing and didn't see any there

That only controls the default prices that get added when you create a new ticket.
EE currently (even in current master) has no UI for editing default tickets.

**Comment by garthkoyle** [comment]
Okay, so if there was a bug in the tickets not showing for new events, then we can close this issue and resume testing.

**Comment by tn3rb** [comment]
yes the new tickets not showing was fixed in https://github.com/eventespresso/event-espresso-core/issues/2999 and was actually a super subtle issue in the `EE_Ticket` class in EE core
